src/sass/_common/external/gridle/_mixins.scss

```diff
@@ -290,14 +290,16 @@
 	@if gridle_has_state($stateName) {
 		// extend the "at-root" version for the .state-{stateName} classes
 		@if $states-classes and str-index(unquote("#{&}"), unquote('.state-#{$_gridle_current_stateName}')) {
-			// @at-root (without: media) { // seems not needed here, cause issues with libsass
+			@at-root (without: media) {
```

Comment:
doesnt this have impact on the styles?

Reply:
@ali-hosseini-deriv I uncommented this because of the error:
>"You may not @extend selectors across media queries."

`@at-root (without: media)` helps us move this `@extend` outside of the `@media`, right where it's supposed to be. Styles look good after this change. I wouldn't use this approach if the classname was regular, not made of sass-variables. :)